### ISSUE TITLE
Travis CI not working

### ISSUE THREAD

**Issue by darrencl** [openmrslab/suspect#166]
Hi @bennyrowland ,

I created a new PR #165 and it seems that the Travis CI (coveralls included) is not triggered anymore for some reason. I logged into Travis and there is an error message: `We are unable to start your build at this time. You exceeded the number of users allowed for your plan. Please review your plan details and follow the steps to resolution.` (not sure if this message is for my account or repo/organisation). 

[This](https://isdown.app/integrations/travis-ci/incidents/77148-some-users-are-facing-issues-when-triggering-a-build-on-travis-ci-platform) might be relevant. I used GitHub action in the past, but never set up Travis CI, so I am not familiar with the pricing/plan etc. It seems that they now require paid plan for unlimited usage and the free plan only lasts for 30-days. 

As you are managing the `openmrslab` organisation, could you please check this?

I'm happy to migrate our CI to GitHub actions if needed. 

Thanks,
Darren

**Comment by bennyrowland** [comment]
@darrencl thanks for letting me know - I haven't had any time to do maintenance of Suspect for a while and this is one of the results. I have heard a lot of people saying good things about GHA so this is probably a good time to make the transition to using that instead of Travis. If you have the time/inclination to take that on, then please feel free to look into it. I have sent you an invite to join the openmrslab organisation which should give you the permissions you need. If you don't fancy it, then I will try and find some time but no guarantees of when it will be, sorry.

**Comment by darrencl** [comment]
@bennyrowland Thank you for your response! I have accepted the invite and will try to find a time to migrate the CI to GHA.

I would like to put the PyPi deployment into GHA, like what you did with Travis CI. To do this, I believe I would need to be added as a maintainer or so in the [Suspect's PyPi project page](https://pypi.org/project/suspect/)? 

I don't have experience in deploying/publishing a package into PyPi, so I am wondering how to test if the CI deployment job actually works, without actually publishing anything? I am looking into https://test.pypi.org/, which might be what we need. However, I just found there is a project called suspect in the test PyPi (https://test.pypi.org/project/suspect/), which is definitely not yours. Maybe we can discuss this further once I submit a PR on GHA.
